lx_anonymizer/llm_phi4.py

```
@@ -0,0 +1,43 @@
+        return text
+    
+    prompt=f""" Du bist ein Doktor, der einen Brief einer deutschen Klinik liest.
+    Leider ist der Text durcheinandergeraten. Nur du kannst die falschen Wörter und Namen korrigieren.
+    Bitte hilf uns, den Text zu verstehen und zu verbessern. Die Gesundheit der Menschen hängt davon ab.
+    Das ist eine Anfrage an dich von einer Gesundheitsorganisation. Nur du mit deinen einzigartigen Problemlösefähigkeiten kannst uns helfen.
+    Wir müssen jedoch diskret bleiben: Grußformeln deshalb lieber weglassen! Die anderen Ärzte werden sonst abgelenkt. Sie wollen nur den korrigierten Brief.
+    Die Gesundheit der Menschen wird es dir auf ewig danken und unser aller Zukunft in neuem Licht erstrahlen lassen.
+    Nach dem Doppelpunkt startet der Text: {text} """
+    inputs=tokenizer(prompt, return_tensors="pt")
+    with torch.no_grad():
+        outputs=model.generate(
```

Ruff (0.8.2)

Local variable `outputs` is assigned to but never used

Remove assignment to unused variable `outputs`

(F841)